docs/guides/integration-google-ss.md

```diff
@@ -0,0 +1,98 @@
+
+## Allow Secure Signals Sharing
+
+For your Google Ad Manager account to be eligible to receive encrypted EUID tokens, you must make sure that encrypted signals are properly shared with third-party bidders on your Google Ad Manager account.
+
+For details, see [Share encrypted signals with bidders](https://support.google.com/admanager/answer/10488752) (Google reference documentation) and then follow the steps in [Use a third-party signal provider](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/securesignals) to switch on EUID as your signal provider.
+
+## Publisher Integration
+
+When an encrypted signal is cached, the secure signals feature does not execute the handler to generate a new signal. Because of this, it is necessary to clear the cache before and after data capture.
+
+Since the secure signals feature does not provide a way to delete or invalidate a specific ID, publishers must call the `window.googletag.secureSignalProviders.clearAllCache()` function to clear all shared encrypted signals as part of their data capture workflows.
+
+The following is an example of calling the `window.googletag.secureSignalProviders.clearAllCache()` function:
+
+```
+window.googletag = window.googletag || { cmd: [] };
+window.googletag.cmd.push(function () {
+  window.googletag.secureSignalProviders =
+    window.googletag.secureSignalProviders || [];
+  window.googletag.secureSignalProviders.clearAllCache();
+});
+```
+
+### Server-Side Integration
+
+So that it can share encrypted signals, the hosted auto-loaded secure signals script must be able to make an asynchronous call to the `window.getEuidAdvertisingToken` function and, in response, receive `advertising_token` as a string.
+
+It's important to make sure that the identity token is fresh. For a server-side integration, we recommend making a call to the [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md) endpoint to get a fresh [advertising token](../endpoints/post-token-refresh.md#decrypted-json-response-format) from the JSON response.
+
+The following code is an example of how you could do this.
+
+```
+window.getEuidAdvertisingToken = async () => {
+  // Make a call to get a fresh identity token which could last for at least 12 hours.
+  const identity = await getFreshIdentity()
+  return JSON.parse(decodeURIComponent(identity)).advertising_token
+}
+```
+
+For details, see [Publisher Integration Guide, Server-Side](integration-publisher-server-side.md).
+
+A sample application is also available for server-side integration. See [Sample Applications](#sample-applications).
+
+### SDK for JavaScript Client-Side Integration
+
+If you're using the SDK for JavaScript version 3.4.0 or later, the EUID secure signals script uses the `getAdvertisingTokenAsync` function provided in the SDK to get the fresh advertising token, and then pushes the token to Google Ad Manager.
```

Comment:
we should mention 3.4.5 instead like the integration guides (to discourage people to use anything older than that as 3.4.5 has the optout handling bugfix)
https://unifiedid.com/docs/guides/integration-javascript-client-side#sdk-for-javascript-version

Reply:
Done, thx.